### PR TITLE
Update constants.py

### PR DESCRIPTION
**Pull request by TheDarkDrake**
Do check them ><

### CODE REVIEW

File: sugaroid/brain/constants.py [hunk 1]
```diff
@@ -527,17 +537,21 @@
...
 
 HI_RESPONSES = [
     "Hi there",
     "Hola amigo!",
-    "Hello, welcome to sugaroid",
-    "Hey, how can I help you?",
-    "Hey dude, how can I help you?",
-    "Hey there",
+    "Hey, how can I be of help?",
+    "Hey bro, how can I help you?",
```
Comment (srevinsaju) [inline sugaroid/brain/constants.py:553]: need to use a gender neutral word instead of `bro`

**Review by srevinsaju — CHANGES_REQUESTED** [review]
Looks good, better responses than before

File: sugaroid/brain/constants.py (hunk 1 above)
Comment (TheDarkDrake) [reply sugaroid/brain/constants.py:553]: ok

File: sugaroid/brain/constants.py [hunk 2]
```diff
@@ -548,4 +562,6 @@
     "TTYL!",
     "Catch you later!",
     "Bye!",
+    "see you later!",
+    "addio",
```
[the remark this answers is not in the document]
Comment (TheDarkDrake) [reply sugaroid/brain/constants.py:566]: adieu --> goodbye(french)
addio --> bye(italian)

[the remark this answers is not in the document]
Comment (srevinsaju) [reply sugaroid/brain/constants.py:566]: Ohh, it would be nice to have both then :smile: if posible

[the remark this answers is not in the document]
Comment (TheDarkDrake) [reply sugaroid/brain/constants.py:566]: indeed, I added that now

**Review by srevinsaju — APPROVED** [review]

**Pull request merged**